drivers/gpio/gpio_bd8lb600fs.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 SILA Embedded Solutions GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file Header file for the BD8LB600FS driver.
+ */
+
+#ifndef ZEPHYR_DRIVERS_GPIO_GPIO_BD8LB600FS_H_
+#define ZEPHYR_DRIVERS_GPIO_GPIO_BD8LB600FS_H_
+
+#include <zephyr/kernel.h>
+
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/spi.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Configuration data */
```

Comment:
Please move the config and data structures to the `.c` file. This header can be removed.

Reply:
Okay for me. The only issue I have is that I am in general a lazy person and I basically just copied and modified the driver of MCP23S17 (https://github.com/zephyrproject-rtos/zephyr/blob/d56ed39e209d63f95593218f95fd04ea97d5bea6/drivers/gpio/gpio_mcp23s17.h). Therefore, I assume it is also wrong for the MCP23S17?

Reply:
I would say so, yea

Reply:
done

Reply:
Applied the same changes to the origin: #54814